style_editor.html

```diff
@@ -3220,6 +3221,12 @@
 function AddPreon() {
   blade.addEffect(EFFECT_PREON, 0.0);
 }
+function AddBattery() {
```

Comment:
We don't need these sort of functions for a drop-down though.
We'll just call addEffect() directly.

Reply:
Not sure what this means.

Reply:
We don't need this function.

Reply:
directly as in at the button assignment?
`    <input name=battery type=button value="Batt Level" onclick="blade.addEffect(EFFECT_BATTERY_LEVEL, 0)"; >
`
Need clarity on actuation button or no actuation button.
Sounds like yes if they can be in a drop down menu to save real estate?

Reply:
No, I mean that we are going to want a function that finds the currently selected effect from the drop-down, then calls blade.addEffect on that.